client/src/game/star.js

```diff
@@ -6,6 +6,19 @@ class Star extends EventEmitter {
 
   static culling_margin = 16
 
+  /*
+    Defines what zoompercentage correspond to what
+    depth level.
+    This is something you can tinker with
+    Potentially even make user defined.
+  */
+  static zoomLevelDefinitions = {
+    1: 150,
```

Comment:
Isn't this something we'd **always** want to be drawn? So set to `0` or removed?

Reply:
Yep, setting it to `0`